src/components/leaderboard/LeaderboardRow/style.module.scss

```
@@ -0,0 +1,69 @@
+@use 'src/styles/vars.scss' as vars;
+
+.row {
+  align-items: center;
+  background-color: var(--theme-surface-1);
+  display: grid;
+  font-size: 15px;
+  gap: 32px;
+  grid-template-columns: auto auto 2fr 1fr;
+  height: 64px;
+  padding: 0 32px;
+
+  &[data-style='odd'] {
+    background-color: var(--theme-background);
+  }
+}
+
+.position {
+  font-size: 20px;
+  font-weight: bold;
+  width: 40px;
+}
+
+.profilePicture {
+  border-radius: 50%;
+}
+
+.nameRank {
+  display: grid;
+  gap: 32px;
+  grid-template-columns: 1fr 1fr;
+}
+
+.name,
+.points {
+  font-weight: 500;
+}
+
+.rank {
+  color: var(--theme-text-on-surface-1);
+  opacity: 0.5;
+  text-align: center;
+}
+
+.points {
+  text-align: right;
+}
+
+@media (max-width: vars.$breakpoint-md) {
+  .row {
+    gap: 16px;
```

```suggestion
    gap: 1rem;
```
nit: we try to use `rem` values in this project for any values greater than `4px` for better responsiveness on zoom in/out and user font size preferences